### ISSUE TITLE
[en] false alarm MASS_AGREEMENT

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#1656]
> The e-commerce site

<img width="479" alt="Bildschirmfoto 2019-06-23 um 19 05 38" src="https://user-images.githubusercontent.com/37363/59979577-f7dec080-95e9-11e9-824c-80fdde42ea2f.png">

**Comment by MikeUnwalla** [comment]
I corrected the error for 'e-commerce site', but the general problem remains, thus, I did not close the issue.

If a word can be both a noun and a verb, and if the verb is a transitive verb, then we can parse the sentence as DT + noun cluster.

But, the rule will always give FP for words that are both nouns and verbs, if the verb is intransitive. Refer to the 'oil leak' example.

I am not sure about the best way to deal with the general problem. Possibly, just change the message to say why it is not possible to disambiguate in these cases.

For a few examples of how the transitivity of a verb has an effect on how we parse text, refer to the first table in www.simplified-english.co.uk/analysis.html (not all the examples are about transitivity).
